types/uppercamelcase/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for uppercamelcase
+// Type definitions for uppercamelcase 3.0
 // Project: https://github.com/samverschueren/uppercamelcase
 // Definitions by: York Yao <https://github.com/plantain-00>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function upperCamelCase(...args: string[]): string;
 export = upperCamelCase;
-declare namespace upperCamelCase { }
```

Comment:
Thank you for your review!

I was not aware that removing the namespace disables the `import * as ...` syntax. But to be honest, I'm not even sure, whether its worth it to keep this syntax. This syntax won't work in a native module-aware ES6 environment because ES6 module syntax doesn't allow you to export functions directly. So this is purely cosmetic and is even mentioned in the module-function template:
```
/*~ Note that ES6 modules cannot directly export callable functions.
 *~ This file should be imported using the CommonJS-style:
 *~   import x = require('someLibrary');
 *~
 *~ Refer to the documentation to understand common
 *~ workarounds for this limitation of ES6 modules.
 */
```

Regarding the compilation, I don't really understand what you mean. I've set up a new project and tested it out, the compilation output looks the same regardless of whether the namespace is present or not:
```js
"use strict";
exports.__esModule = true;
var upperCamelCase = require("uppercamelcase");
console.log(upperCamelCase('-asdas-asd'));
```